openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -1572,16 +1572,17 @@ def assign_parameters(self, topology, system):
         """
         pass
 
-    def postprocess_system(self, topology, system, **kwargs):
+    def postprocess_system(self, system, topology, **kwargs):
```

Comment:
Seems like this belongs in another PR?

Reply:
Agreed. Is this intentional? Is it necessary? I think it should be removed from this PR regardless, but should it make its way in in another PR?

Reply:
I am also uncomfortable with this slipping in here, so I am overwriting these changes. If this was intentional it can be placed elsewhere but I suspect it was added accidentally last fall

Reply:
Oops, yes, that belongs elsewhere... I suspect that `LibraryChargeHandler` took care of this. Thanks for spotting this.